packages/grafana-schema/src/schema/legend.gen.ts

```diff
@@ -3,14 +3,32 @@
 // It is currenty hand written but will serve as the target for cuetsy
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+/**
+ * @public
```

Comment:
Will we always generate these?

Reply:
Ideally not needed -- ideally we just replace the [api-extractor.json](https://github.com/grafana/grafana/blob/schema-package/packages/grafana-schema/api-extractor.json) file with one that says *everyting* is public

Reply:
> Does CUE allow providing comments on the generated code?

It gives [structured access](https://pkg.go.dev/cuelang.org/go@v0.4.0/cue#Value.Doc) to the comments associated with the AST node(s) that comprise a logical value (by virtue of the same structure that allows us to know with certainty that we can always trace validation errors back to the line declaring them), but does no additional parsing _within_ the comment object. We can do our own custom parsing of comment blocks and take action, though just a dumb passthrough of the literal comment text seems sufficient from what i see in this PR.

> Ideally not needed -- ideally we just replace the api-extractor.json file with one that says everyting is public

If this means that `@public` is just _implied_ on every structure by any TS reading this package, then 👍 . However, it might be trouble if "api-extractor" (i have no idea what it is/does) actually generates comments in the source files, as then we'd have the above comments generated by CUE, AND those comments.